libethash-cuda/ethash_cuda_miner.cpp

```diff
@@ -63,17 +63,12 @@ ethash_cuda_miner::search_hook::~search_hook() {}
 
 ethash_cuda_miner::ethash_cuda_miner()
 {
-	int devicesCount = getNumDevices();
-	m_light = new hash64_t*[devicesCount];
-	for(int i = 0; i < devicesCount; i++)
+	int deviceCount = getNumDevices();
+	m_light = std::vector<hash64_t*>(deviceCount);
+	for(int i = 0; i < deviceCount; i++)
```

Comment:
Not needed. vector's constructor will init all elements with default values, in this case with `nullptr`.